crates/subspace-node/res/chain-spec-raw-devnet.json

```diff
@@ -16,7 +16,7 @@
   "protocolId": "subspace-devnet",
   "properties": {
     "dsnBootstrapNodes": [
-      "/dns/bootstrap-0.devnet.subspace.network/tcp/30533/p2p/12D3KooWJgLU8DmkXwBpQtHgSURFfJ4f2SyuNVBgVY96aDJsDWFK"
+      "/dns/bootstrap-0.devnet.subspace.network/udp/30533/quic-v1/p2p/12D3KooWJgLU8DmkXwBpQtHgSURFfJ4f2SyuNVBgVY96aDJsDWFK"
```

Comment:
Shouldn't we have both QUIC and TCP, just with QUIC being the first one here?

Reply:
I would use only QUIC at least in the beginning to test QUIC. After gemini-3g we will enable both protocols.